### DISCUSSION TITLE
Hi temperature hot end Firmware

### DISCUSSION THREAD

**Discussion by JCRodzDom** [Jyers/Marlin#1384] [Ideas]
Can we have the firmware set for a higher temperature. It only goes up to 260. Im printing some Nylons with my Ender 3 v2 and it has all the hardware to do it, but cant fix the firmware my self. I think it should be set to 310 as a basic for everyone. 
Please advice.

**Comment by JCRodzDom** [comment]
Guys I tried to compile the firmware to a higher temp but the out put was an .hex not a .bin 
And the printer will not take it.
What im doing wrong?

**Reply by Cal-LF** [reply]
Reference #1335 to follow the compile instructions given to another user. 
If you need assistance I will guide you to successful compiled .bin
I do not know which settings need to be changed for your higher temp requirements. If you can make those changes in the config files, we can get it compiled.

**Comment by JCRodzDom** [comment]
Hi Guys, so this is what i did, let me know where i get lost.
1) download the Marlin - 1.3.5.b Source Code
2) went to configuration files folder and open folder Jyyer's Config
3) at Jygers Config folder, I copied two files: Configuration and Configuration_adv
4) Went back to Marlin folder and paste copied files to over write the existing.
5) Open Visual studio code , and open new project.
6) at Visual Studio, found the Configuration_h file and make my changes like change maximum temp to 310 degrees.
7) Compile the work successfully, but cant find the .bio folder neither the .bin as per many videos.
Please let me know what im missing.

**Comment by JCRodzDom** [comment]
ok so I just find a video that shows on Visual Studio Code that after all that you have to go to File and hit Open Folder ...
this operation for some reason creates the .bio folder and the .bin files.
So it works out good.
Now , i changed the program to 310 degrees but it is only giving me 295 degrees max.
Any other parameters the i need to change ?

**Comment by dfgyhhju** [comment]
hi friend.you can share the firmware for 300c. I can't compile
